src/components/ResultListItem.js

```diff
@@ -109,7 +109,7 @@ class ResultListItem extends React.Component {
                     <div>
                         {this.nMoreRelatedServices} more{' '}
                         {this.nMoreRelatedServices == 1 ?
-                            'service' : 'services'}...
+                            'service' : 'services'}…
```

Comment:
I don't think I've seen this construct before, is it a React thing? Is it meant to be three periods or an ellipsis?